tests/console/a2ps.pm

```diff
@@ -2,15 +2,15 @@ use strict;
 use base "consoletest";
 use testapi;
 
-# a2ps is available by default only in openSUSE*
-
 sub run() {
     become_root;
+    assert_script_run("systemctl stop packagekit.service");
```

Comment:
you shouldn't do that - the a2ps test always runs behind consoletest_setup and is meant to be.